Dockerfile

```diff
@@ -30,7 +30,7 @@ COPY install/ install/
 # Build
 RUN ./hack/build.sh
 
-FROM registry.access.redhat.com/ubi8/ubi:latest
+FROM registry.access.redhat.com/ubi9/ubi:latest
```

Comment:
not an issue as long as we build a statically linked binary AFAIK, but I think it's better to update the builder image to centos:stream9 as well